crates/bin/pindexer/Cargo.toml

```diff
@@ -45,6 +45,7 @@ tokio = {workspace = true, features = ["full"]}
 serde_json = {workspace = true}
 serde = { workspace = true, features = ["derive"] }
 sqlx = { workspace = true, features = ["chrono", "postgres"] }
+hex = "0.4"
```

Comment:
Nit: the `hex` dependency, if required, should be formatted as `hex = { workspace = true }`, so that there's only one version of `hex` throughout the repo to manage. However, I'm not convinced the dep addition is really necessary....